app/controllers/admin/content_controller.rb

```diff
@@ -27,6 +27,21 @@ def new
     new_or_edit
   end
 
+  def merge_with
+    @article = Article.new(params[:article])
+    @merged_article = Article.find(params[:merge_with])
```

Comment:
Do you need to handle the articles comments? Do you need to further restrict this to the admin or is putting it in this controller sufficient?